### ISSUE TITLE
Characters after $lines:1 remove $lines:1-content

### ISSUE THREAD

**Issue by denchuku** [mrjackphil/obsidian-text-expand#34]
Version: 0.7.4
(Not sure if this resulted from the changes of https://github.com/mrjackphil/obsidian-text-expand/issues/32 )

### What works

    ```expander
    tag:#location
    - [[$filename]] $lines:1 
    ```

Produces:
```
- [[foo]] First line of text in file
- [[bar]] First line of text in file
```

### What doesn't work
Having characters after "$lines:1"...:

    ```expander
    tag:#location
    - [[$filename]] $lines:1 bar
    ```
Produces:
```
- [[foo]] bar
- [[bar]] bar
```

Expected:
```
- [[foo]] First line of text in file bar
- [[bar]] First line of text in file bar
```

**Comment by denchuku** [comment]
This works:
```
- [[$filename]] $lines:1
```
-> "$lines:1" is rendered in output

This doesn't work:
```
- [[$filename]]: $lines:1
```
-> "$lines:1" is not rendered in output

So it's not just characters after "lines:1" but also in other cases...

**Comment by mrjackphil** [comment]
Thanks, @denchuku. Should be fixed in the latest 0.7.5 release.

**Comment by denchuku** [comment]
Perfect! That's great!
